packages/esm-patient-forms-app/src/index.ts

```diff
@@ -9,11 +16,6 @@ function setupOpenMRS() {
     pattern: '.+/ws/rest/v1/form.*',
   });
 
-  messageOmrsServiceWorker({
```

Comment:
Why remove it?

Reply:
Got deleted by mistake while rebasing, returned it in